### PR TITLE
Evidence cpt

### PR DESCRIPTION
**Pull request by BNewing**
This adds in the evidence custom post type, and a page on which to display its posts. It also adds in filter functionality, so you can filter the results.

It makes use of CSS grid and flexbox where supported, but otherwise has fallbacks, which work back to internet explorer 9 (we generally support back to internet explorer 11). 

The filter functionality is dependent on JS, so this is only visible if JS is enabled in the browser. 

There is a lot of GOV.UK Design System styles in the `evidence.scss` file - redundant styles will be removed once we've finalised what the filter should be.

This addresses Trello ticket #651.

### CODE REVIEW

File: web/app/themes/brookhouse/assets/js/evidence.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          $('.evidence__item[data-evidence-type="' + entry + '"]').show();
+          $('.evidence__item[data-evidence-format="' + entry + '"]').show();
+          $('.evidence__item[data-witness-type="' + entry + '"]').show();
+        });
+      }
+    }
+  })
+
+  if ($('#js-evidence-filter').length) {
+    $('#js-evidence-filter').change(function () {
```
Comment (RobLoweMOJ) [inline web/app/themes/brookhouse/assets/js/evidence.js:48]: How come the change listener is on the div container. If it was on the checkbox thats clicked you could target more precise as your TO DO says.

File: web/app/themes/brookhouse/page-evidence-listing.php [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+<div id="primary" class="content-area">
+    <main id="main" class="site-main documents-main">
+        <?php while (have_posts()) : the_post(); ?>
+
+            <h1><?php the_title(); ?></h1>
+            <?php the_content(); ?>
+
+            <?php $query = new WP_Query( array( 'post_type' => 'evidence', 'paged' => $paged ) ); ?>
+
+            <p id="js-turned-off">Please turn JavaScript on in your browser, to enable the filter functionality.</p>
```
[the remark this answers is not in the document]
Comment (BNewing) [reply web/app/themes/brookhouse/page-evidence-listing.php]: Good spot, thank you! Just pushed up a commit to fix this :)

File: web/app/themes/brookhouse/assets/js/evidence.js (hunk 1 above)
Comment (BNewing) [reply web/app/themes/brookhouse/assets/js/evidence.js]: Thank you for the reminder! I've made this more specific now (and removed the TO DO)

File: web/app/themes/brookhouse/inc/taxonomies.php [hunk 3]
```diff
@@ -55,3 +61,47 @@ function brookhouse_create_document_taxonomies()
...
     }
 }
+
+/**
+*
+* Register the taxonomies used in the evidence type section
+*
+* */
+add_action('init', 'brookhouse_create_evidence_taxonomies');
+
```
[the remark this answers is not in the document]
Comment (BNewing) [reply web/app/themes/brookhouse/inc/taxonomies.php:71]: Ok! Could we pair on that :D?

[the remark this answers is not in the document]
Comment (RobLoweMOJ) [reply web/app/themes/brookhouse/inc/taxonomies.php:71]: Sounds good. :D

**Pull request merged**